### ISSUE TITLE
Handling of primary selections depends on "automatic commands" option

### ISSUE THREAD

**Issue by weiss** [hluk/CopyQ#2651]
**Describe the bug**
For primary selections to be [stored in the history][1], it seems the _Run automatic commands on selection_ (`run_selection`) option must be enabled in addition to the _Store text selected using mouse_ (`check_selection`) option.

[1]: https://github.com/hluk/CopyQ/issues/1456

**To Reproduce**
```
copyq config check_selection true
copyq config run_selection false
```
Then select a few items with the mouse, and see how those weren't added to the history:
```
copyq read 0 1
```

**Expected behavior**
I'd expect `copyq read 0 1` to list items selected with the mouse regardless of the `copyq config run_selection` value.

**Version, OS and Environment**
CopyQ Clipboard Manager 6.4.0
Qt: 5.15.8
KNotifications: 5.101.0
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: Debian GNU/Linux 12 (bookworm)

I was able to reproduce the behavior both on Wayland and X11.

**Comment by hanschen** [comment]
Nice find, this explains why primary selection wasn't working for me. Can confirm on:

**Version, OS and Environment**
CopyQ Clipboard Manager 8.0.0
Qt: 5.15.12
KNotifications: 5.115.0
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: Arch Linux

**Comment by hluk** [comment]
Fixed in  #2681 (it will be available in the next release). Thanks for the report!
